### PR TITLE
`wiby clean`

### LINKED ISSUE #79: `wiby clean` command
At the moment, `wiby test` creates a branch called `wiby-[dep name]` in the dependent repos. There is nothing to clean this up - running a new `wiby test` will fail if the branch already exists.

`wiby test` should not fail on existing branch - but that is a separate, larger problem (I'll create a separate issue), but in the meantime - deleting the `wiby-wiby` branch manually on `wiby-test/*` repos is a hassle during development.

`wiby clean` should take the following options:

- `--all` - remove all `wiby-*` branches (by default only remove the current branch, based on the naming as proposed in #78)
- `--dry-run` - show the branches that would be removed
    - alternative: do the dry-run by default and ask the user to confirm the action? With an optional `--force` to skip the confirmation?
    - bonus points: what if the branches contain commits other than ones made by wiby?

### PR DESCRIPTION
**Pull request by dominykas**
Closes #79

### CODE REVIEW

File: lib/clean.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const github = require('./github')
+const gitURLParse = require('git-url-parse')
+const logger = require('./logger')
+
+// setup logger namespace
+const testCommandNamespace = 'wiby:clean'
+const debug = logger(testCommandNamespace)
+
+module.exports = async ({ dependents }, { all, dryRun }) => {
+  // enable log output for test command without DEBUG env
```
Comment (rodion-arr) [inline lib/clean.js:12]:
```suggestion
  // enable log output for clean command without DEBUG env
```

File: lib/clean.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const github = require('./github')
+const gitURLParse = require('git-url-parse')
+const logger = require('./logger')
+
+// setup logger namespace
+const testCommandNamespace = 'wiby:clean'
```
Comment (rodion-arr) [inline lib/clean.js:8]:
```suggestion
const cleanCommandNamespace = 'wiby:clean'
```

File: lib/clean.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+const debug = logger(testCommandNamespace)
```
Comment (rodion-arr) [inline lib/clean.js:9]:
```suggestion
const debug = logger(cleanCommandNamespace)
```

File: test/clean.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  nock.enableNetConnect()
+})
+
+tap.test('wiby.clean()', async (tap) => {
+  tap.test('should check if the wiby branch exists', async (tap) => {
+    nock('https://api.github.com')
+      .get(`/repos/wiby-test/${CONFIG.DEP_REPO}/branches/wiby-wiby`)
+      .reply(404)
+
+    await wiby.clean({ dependents: [{ repository: `https://www.github.com/${CONFIG.DEP_ORG}/${CONFIG.DEP_REPO}` }] }, {})
```
Comment (rodion-arr) [inline test/clean.js:24]: Could you please elaborate what is assertion here? It shouldn't throw exception?

**Review by rodion-arr — APPROVED** [review]
👍🏻

File: lib/clean.js (hunk 3 above)
Comment (dominykas) [reply lib/clean.js]: https://github.com/pkgjs/wiby/pull/81/commits/0108df89afe72267c043b8e6e18d56de39c17584

File: lib/clean.js (hunk 2 above)
Comment (dominykas) [reply lib/clean.js]: https://github.com/pkgjs/wiby/pull/81/commits/0108df89afe72267c043b8e6e18d56de39c17584

File: lib/clean.js (hunk 1 above)
Comment (dominykas) [reply lib/clean.js]: https://github.com/pkgjs/wiby/pull/81/commits/0108df89afe72267c043b8e6e18d56de39c17584

File: test/clean.js (hunk 4 above)
Comment (dominykas) [reply test/clean.js:24]: https://github.com/pkgjs/wiby/pull/81/commits/196d30a1d94ea402a658ab91844d1131d45044b9

**Pull request merged**